pkg/apis/enterprise/v1beta1/common_types.go

```diff
@@ -177,3 +177,26 @@ type IndexAndCacheManagerCommonSpec struct {
 	// Time period relative to the bucket's age, during which the bloom filter file is protected from cache eviction
 	HotlistBloomFilterRecencyHours uint `json:"hotlistBloomFilterRecencyHours,omitempty"`
 }
+
+// ApplicationFrameworkSpec defines the application package remote store repository
+type ApplicationFrameworkSpec struct {
+
+	// Is Application Framework Feature Enabled. 
+	// Default value is False
+	FeatureEnabled string `json:"featureEnabled"`
```

Comment:
Any reason why a boolean wouldn't work here?

Reply:
changed to bool